api/src/main/java/org/openmrs/module/queue/api/search/QueueEntrySearchCriteria.java

```diff
@@ -61,11 +61,15 @@ public class QueueEntrySearchCriteria implements Serializable {
 	
 	private Date startedOnOrBefore;
 	
+	private Date startedOn;
+	
 	private Boolean isEnded = null;
 	
 	private Date endedOnOrAfter;
 	
 	private Date endedOnOrBefore;
 	
+	private Date endedOn;
```

Comment:
If we are expanding the capabilities of QueueEntrySearchCriteria, we should also add the necessary code to support these new search parameters via the REST endpoint.  Can you make the necessary improvements to `QueueEntrySearchCriteriaParser`?